resources/assets/js/split.js

```diff
@@ -0,0 +1,14 @@
+$('#competition_max').change(function() {
+  var waitingRoom = window.location.pathname.split('/')[2];
```

Comment:
Hmmmm, maybe this  is something we should be including in the page? This could break unexpectedly if we change the URL structure (like adding a prefix or something).